README.md

```diff
@@ -1,6 +1,12 @@
 # PaddlePaddle Cloud
 
-## Getting Started
+## Using Command-Line To Submit Cloud Training Jobs
+
+You can find a useful documentation [here](./doc/v1/usage_en.md).
+
+For training with v2 API the documentation is [here](./doc/v2/usage_en.md).
```

Comment:
Maybe we don't need an independent page for v2, because of there only a small difference between v1 and v2 submitting parameters.